ui/pages/send/send-content/add-recipient/domain-input.component.js

```diff
@@ -156,20 +161,12 @@ export default class DomainInput extends Component {
                     this.props.scanQrCode();
                   }
                 }}
-              >
-                <i
-                  className={classnames('fa', {
-                    'fa-times': userInput,
-                    'fa-qrcode': !userInput,
-                  })}
-                  title={t(userInput ? 'close' : 'scanQrCode')}
-                  style={{
-                    color: userInput
-                      ? 'var(--color-icon-default)'
-                      : 'var(--color-primary-default)',
-                  }}
-                />
-              </button>
+                iconName={
+                  userInput ? ICON_NAMES.CLOSE : ICON_NAMES.SCAN_BARCODE
+                }
+                title={t(userInput ? 'close' : 'scanQrCode')}
```

Comment:
Instead of title I think the equivalent prop would be `ariaLabel`
```suggestion
                ariaLabel={t(userInput ? 'close' : 'scanQrCode')}
```